src/main/java/duke/exceptions/DukeException.java

```diff
@@ -0,0 +1,17 @@
+package duke.exceptions;
+
+/**
+ * A RuntimeException that is thrown by Duke.
+ * All exceptions thrown by Duke should be a subclass of DukeException.
+ */
+public class DukeException extends RuntimeException {
+
+    /**
+     * A RuntimeException that is thrown by Duke.
+     *
+     * @param message The message to be displayed when the exception is thrown.
+     */
+    public DukeException(String message) {
+        super(message);
+    }
+}
```

Comment:
Good use of unique exception classes to handle errors!
I especially like that you also made subclasses of this class to specify the different types of error. 👍